es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 **1. Unsupervised Learning cheatsheet**
 
-&#10230;
+&#10230; Aprendizaje no Supervisado
 
 <br>
 
 **2. Introduction to Unsupervised Learning**
 
-&#10230;
+&#10230; Introducción al Aprendizaje no Supervisado
 
 <br>
 
 **3. Motivation ― The goal of unsupervised learning is to find hidden patterns in unlabeled data {x(1),...,x(m)}.**
 
-&#10230;
+&#10230; Motivación - El objetivo del aprendizaje no supervisado es encontrar patrones ocultos en datos no etiquetados {x(1),...,x(m)}.
 
 <br>
 
 **4. Jensen's inequality ― Let f be a convex function and X a random variable. We have the following inequality:**
 
-&#10230;
+&#10230; Desigualdad de Jensen - Sea f una función convexa y X una variable aleatoria. Tenemos la siguiente desigualdad:
 
 <br>
 
 **5. Clustering**
 
-&#10230;
+&#10230; Clusterización (Agrupamiento)
```

Comment:
"Clusterización" is an anglicism which is not accepted by the RAE (although "clúster" is). I would translate it as "_Agrupamiento (clustering en inglés)_". Furthermore, later in the text, you don't use "clusterización" any more and, instead, you use "_agrupamiento_".